collectors/template/collector.js.template

```diff
@@ -0,0 +1,96 @@
+/* -----------------------------------------------------------------------------
+ * @copyright (C) 2019, Alert Logic, Inc
+ * @doc
+ *
+ * {{type}} class.
+ *
+ * @end
+ * -----------------------------------------------------------------------------
+ */
+'use strict';
+
+const moment = require('moment');
+const PawsCollector = require('@alertlogic/paws-collector').PawsCollector;
+
+
+const typeIdPaths = [
+    // enter your type paths in the form { path: ['myKey'] }
+];
+
+const tsPaths = [
+    // enter your timestamp paths in the form { path: ['myKey'] }
+];
+
+
+class {{Type}}Collector extends PawsCollector {
+    constructor(context, creds) {
+        super(context, creds, '{{type}}');
+    }
+    
+    pawsInitCollectionState(event, callback) {
+        const startTs = process.env.paws_collection_start_ts ? 
+                process.env.paws_collection_start_ts :
+                    moment().toISOString();
+        const endTs = moment(startTs).add(this.pollInterval, 'seconds').toISOString();
+        const initialState = {
```

Comment:
Please add some comments describing implementation details. For example here add something like
```
// TODO: define initial collection state specific to your collector.
// You will get this state back with each invocation
```